drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -6847,18 +6847,10 @@ Error RenderingDeviceVulkan::_draw_list_render_pass_begin(Framebuffer *framebuff
 	render_pass_begin.pNext = nullptr;
 	render_pass_begin.renderPass = render_pass;
 	render_pass_begin.framebuffer = vkframebuffer;
-	/*
-	 * Given how API works, it makes sense to always fully operate on the whole framebuffer.
-	 * This allows better continue operations for operations like shadowmapping.
-	render_pass_begin.renderArea.extent.width = viewport_size.width;
+    render_pass_begin.renderArea.extent.width = viewport_size.width;
```

Comment:
```suggestion
	render_pass_begin.renderArea.extent.width = viewport_size.width;
```
You need to have tabs for indentation in your editor